game_event/tests.py

```diff
@@ -265,3 +283,14 @@ def test_fail_add_player_to_event_already_in_it(self, client, saved_game_event,
         assert response.status_code == 200
         with pytest.raises(IntegrityError):
             response = client.post(url, {'answer': 'yes'})
+
+    def test_event_deletion_on_last_player_leave(self, client, saved_game_event, saved_game_event_player):
+        count_game_events_before = GameEvent.objects.count()
+        client.force_login(saved_game_event_player.player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
```

Comment:
We should introduce new variables only if they provide any added value. In this case, you ca just use the variable you already have.
```suggestion
        url = reverse('remove_from_event', args=[saved_game_event.id])
```